docs/configuration/hostbuilder.md

```diff
@@ -249,7 +249,7 @@ public interface IConfigureMarten
 <sup><a href='https://github.com/JasperFx/marten/blob/master/src/Marten/MartenServiceCollectionExtensions.cs#L734-L745' title='Snippet source file'>snippet source</a> | <a href='#snippet-sample_iconfiguremarten' title='Start of snippet'>anchor</a></sup>
```

Comment:
@AlexZeitler, could you update the code sample located in this line? Our snippets are automatically generated from code using [MarkdownSnippets](https://github.com/SimonCropp/MarkdownSnippets). Because of that, if you don't change them, the markdown snippet will be automatically overridden on the build.

Reply:
@oskardudycz will fix it this evening or at the latest over the weekend.

Reply:
@oskardudycz Done